python/scylla/_rust/statement.pyi

```diff
@@ -1,2 +1,16 @@
+from .enums import Consistency, SerialConsistency
+from .execution_profile import ExecutionProfile
+
 class PreparedStatement:
-    pass
+    def set_execution_profile(self, profile: ExecutionProfile) -> PreparedStatement: ...
+    def unset_execution_profile(self) -> PreparedStatement: ...
+    def get_execution_profile(self) -> ExecutionProfile | None: ...
+    def set_consistency(self, c: Consistency) -> PreparedStatement: ...
+    def unset_consistency(self) -> PreparedStatement: ...
+    def get_consistency(self) -> Consistency | None: ...
+    def set_serial_consistency(self, sc: SerialConsistency) -> PreparedStatement: ...
+    def unset_serial_consistency(self) -> PreparedStatement: ...
+    def get_serial_consistency(self) -> SerialConsistency | None: ...
+    def set_request_timeout(self, timeout: float) -> PreparedStatement: ...
+    def unset_request_timeout(self) -> PreparedStatement: ...
+    def get_request_timeout(self) -> float | None: ...
```

Comment:
It is surprising that those methods return new statement, instead of modifying it. The `set/unset` terminology doesn't fit that approach.
- Why did you chose that approach?
- If we go with that approach, dont you think that `with/without` is better terminology than `set/unset`?